tests/integration_test/run_integration_test.py

```diff
@@ -0,0 +1,142 @@
+                            fail("[FAIL] Graphs are isomorph, is_isomorph(G, H) did not detect it")
+                        error_count += 1
+                        are_isomorph_result = False
+                    else:
+                        if console_pass:
+                            passed("Graphs are isomorph")
+
+                    # If amount of isomorphisms in solution is -1, there is no answer known
+                    if amount_isomorph_expected != -1:
+                        if amount_isomorph_actual != amount_isomorph_expected:
+                            if console_fail:
+                                fail("[FAIL] Amount of isomorphisms should be " + str(amount_isomorph_expected) + ", not " + str(amount_isomorph_actual))
+                            error_count += 1
+                            amount_isomorph_result = False
+                        else:
+                            if console_pass:
+                                passed("Amount of isomorphisms is: " + str(amount_isomorph_actual))
+                # Graphs should not be isomorphic
+                else:
+                    if are_isomorph_actual:
+                        if console_fail:
+                            fail("[FAIL] Graphs are not isomorph, is_isomorph determined they were")
+                        error_count += 1
+                        are_isomorph_result = False
+                    else:
+                        if console_pass:
+                            passed("Graphs are not isomorph")
+            else:
+                are_isomorph_expected = '-'
+                are_isomorph_result = '-'
+                amount_isomorph_expected = '-'
+                amount_isomorph_result = '-'
+
+            are_isomorph_time = round((end_isomorph - start_isomorph), 3)
+            amount_isomorph_time = round((end_amount_isomorphisms - start_amount_isomorphisms), 3)
+
+            if console_pass or console_fail:
+                print("Processing time is_isomorph(G, H): " + str(are_isomorph_time) + " s")
+                print("Processing time amount_of_isomorphisms(G, H): " + str(amount_isomorph_time) + " s")
+                print('')
+
+            if write_to_csv:
+                csv_graph_result = [filename, i, j,
+                                    are_isomorph_expected, are_isomorph_actual, are_isomorph_result, are_isomorph_time,
+                                    amount_isomorph_expected, amount_isomorph_actual, amount_isomorph_result, amount_isomorph_time]
+                write_csv_line(csv_filepath, csv_graph_result)
+
+if console_pass or console_fail:
+    print('-------------------------------')
+    print("Statistics of branching test:")
+    print('-------------------------------')
+    print("Amount of graphs tested: " + str(graph_count))
+    print("Amount of graphs with failed results: " + str(error_count))
+    print('')
+    if error_count > 0:
+        fail('TEST FAILED')
```

Comment:
Integration test is handiger in de console. Als er nog andere logging aanstaat van andere individuele tests bijv.

Reply:
Ik zal het aanpassen, maar in principe is de integratie 1 test, die maakt geen gebruik van andere individuele testen. Dat is juist jouw PR